rust-toolchain.toml

```
@@ -1,9 +1,5 @@
 [toolchain]
-# Note: we are stuck on Rust version 1.58 for the time being until
-# this issue is fixed:
-#   https://github.com/rust-lang/rust/issues/97117
-# Since it blocks the ARM64 build of InfluxDB.
-channel = "1.58"
+channel = "1.62"
```

A couple of us ran into an issue with rustup:
https://github.com/rust-lang/rustup/issues/3061 

As a workaround, could we specify `1.62.1` _or if the cross-build is okay on `1.63`_ maybe we could bump all the way up?

```suggestion
channel = "1.62.1"
```

n.b. if we explicitly set `1.62.1` then the Dockerfile should probably match.

---

Whew, time flies. I hadn't realized that 1.63 was out. I'm sure it will work; however, could we stick with 1.62.1 right now? I've got the cross-builder using [1.62.1](https://github.com/influxdata/edge/blob/a6d0dbfb2e0f2313cdf67ade0693047aa0690f94/dockerfiles/cross-builder/install-rust.sh#L7), and it'll take at least 6 PRs in `influxdb` and other repositories to get it updated. :sweat_smile: